include/net/gptp.h

```diff
@@ -207,6 +206,25 @@ struct gptp_phase_dis_cb {
 	gptp_phase_dis_callback_t cb;
 };
 
+/**
+ * @brief ClockSourceTime.invoke function parameters
+ *
+ * Parameters passed by ClockSourceTime.invoke function.
+ */
+struct gptp_clk_src_time_invoke_params {
```

Comment:
This struct looks like there could be holes in it. You could check that with `pahole -C gptp_clk_src_time_invoke_params zephyr.elf` command. In fedora the pahole program can be found in dwarves package.

Reply:
I actually used `pahole -P zephyr.elf` - it is supposed to find all structs with holes that can actually be packed. I assumed it is all that can be done, but I will try doing it the way you described, thanks!

Reply:
ok, no issues then if you already paholed it :)